CRM/Civalpa/Form/CivalpaSettings.php

```diff
@@ -0,0 +1,146 @@
+            "textLineWidth", // field name
+            "Text line width", // field label
+        );
+        $this->add(
+            "checkbox", // field type
+            "useTextRule", // field name
+            "Use text wrap rule", // field label
+        );
+        $this->add(
+            "text", // field type
+            "htmlLineWidth", // field name
+            "HTML line width", // field label
+        );
+        $this->add(
+            "checkbox", // field type
+            "useHtmlRule", // field name
+            "Use HTML wrap rule", // field label
+        );
+        $this->addButtons(
+            [
+                [
+                    'type' => 'submit',
+                    'name' => ts('Update settings'),
+                    'isDefault' => true,
+                ],
+            ]
+        );
+
+        $this->setTitle(ts('Civalpa Settings'));
+
+        parent::buildQuickForm();
+    }
+
+    /**
+     * If your form requires special validation, add one or more callbacks here
+     */
+    public function addRules() {
+        $this->addFormRule(array('CRM_Civalpa_Form_CivalpaSettings', 'configValidator'));
+    }
+
+    /**
+     * Validate configuration
+     *
+     * @param array $values Submitted values
+     *
+     * @return array|bool
+     */
+    public function configValidator($values) {
+        $errors = [];
+        if (!is_numeric($values["textLineWidth"]) || intval($values["textLineWidth"], 10) < 1) {
+            $errors["textLineWidth"] = ts("Only positiv integer values allowed.");
+        }
+        if (!is_numeric($values["htmlLineWidth"]) || intval($values["htmlLineWidth"], 10) < 1) {
+            $errors["htmlLineWidth"] = ts("Only positiv integer values allowed.");
+        }
+        return empty($errors) ? true : $errors;
+    }
+
+    /**
+     * Post process form
+     *
+     * @throws CRM_Core_Exception
+     */
+    public function postProcess() {
+        $newConfig = [
+            "debug-mode" => $this->_submitValues["debugMode"],
+            "text-line-width" => [
+                "use" =>  (bool)($this->_submitValues["useTextRule"]),
+                "value" => intval($this->_submitValues["textLineWidth"], 10),
+            ],
+            "html-line-width" => [
+                "use" =>  (bool)($this->_submitValues["useHtmlRule"]),
+                "value" => intval($this->_submitValues["htmlLineWidth"], 10),
+            ],
+        ];
+        if ($newConfig !== $this->config->get()) {
+            if (!$this->config->update($newConfig)) {
+                CRM_Core_Session::setStatus(ts("Error while updating the config"), E::SHORT_NAME, "error");
+            } else {
+                CRM_Core_Session::setStatus(ts("The config has been updated."), E::SHORT_NAME, "success", [ "expires" => 5000, ]);
+            }
+        } else {
+            CRM_Core_Session::setStatus(ts("The config is not changed."), E::SHORT_NAME, "info", [ "expires" => 5000, ]);
```

Comment:
This `info` here that your config was not changed, could be a little misleading.
I know an `info` is not an `error`, but maybe we could give some more reassurance that everything is fine.

I think of:
"The config is not changed. No update was needed."